pkgs/applications/networking/msmtp/default.nix

```diff
@@ -20,7 +20,7 @@ in stdenv.mkDerivation rec {
     ./paths.patch
   ];
 
-  buildInputs = [ gnutls gsasl libidn2 ]
+  buildInputs = [ gnutls gsasl libidn2 texinfo ]
```

Comment:
I think `textinfo` should go to `nativeBuildInputs`

Reply:
Done